### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by viabadia**

### CODE REVIEW

File: staff/victor-abadia/cv/index.html [hunk 1]
```diff
@@ -5,17 +5,18 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Nomen Nescio - Curriculum Vitae</title>
+    <link rel="stylesheet" type="text/css" href="style.css" media="all" />
 </head>
 <body>
-        <nav id="header">
+        <section id="header">
```
Comment (manuelbarzi) [inline staff/victor-abadia/cv/index.html:11]: AVOID using ids for identifying blocks / elements, do not use them if it's not strictly required by any other well-explained reason

File: staff/victor-abadia/cv/index.html [hunk 2]
```diff
@@ -5,17 +5,18 @@
...
             <h1>Nomen Nescio</h1>
             <p>Age: 07/07/1987</p>
             <p>e-mail: hello@nomennescio.com</p>
             <p>Phone: +34 937777777</p>
             <p>Adress: c/Noname, 1 - 08080 Barcelona</p>
             
-        </nav>             
-        <nav id="content">
+        </section>
```
Comment (vior1992) [inline staff/victor-abadia/cv/index.html:18]: Maybe correct the tabulation?

File: staff/victor-abadia/cv/style.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
+body {
+    color: #313131;
+    background-color: skyblue;
+    border: 4px solid black;
+    margin: 55px;
+}
+a:link {
+    color: red
```
Comment (vior1992) [inline staff/victor-abadia/cv/style.css:8]: You forgot a semi-colon

File: staff/victor-abadia/cv/style.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    margin: 20px;
+}
+.box{
+    border-color:blue;
+}
+#footer  {
+    text-align: center;
+}
+flexbox
+@media (max-width: 600px) {
```
Comment (manuelbarzi) [inline staff/victor-abadia/cv/style.css:39]: your rule (media) is projected as desktop-first, that is, it is fired when width is less-or-equal to max-width. the challenge is to make it the inverse: mobile-first. arrange styles to make it fully that, mobile-first.

**Comment by gbaste** [comment]
Please review pull request and apply the proper changes if any

**Pull request merged**